packages/lesmis/src/views/DashboardReportTabView.js

```diff
@@ -26,11 +26,27 @@ const DashboardSection = styled(FlexCenter)`
   min-height: 31rem;
 `;
 
+const setDefaultDashboard = (data, setSelectedDashboard) => {
+  const dashboardNames = Object.keys(data);
+
+  if (dashboardNames.includes(DEFAULT_DASHBOARD_GROUP)) {
+    setSelectedDashboard(DEFAULT_DASHBOARD_GROUP);
+  } else {
+    setSelectedDashboard(dashboardNames[0]);
+  }
+};
+
 export const DashboardReportTabView = ({ entityCode, TabSelector }) => {
   const [selectedYear, setSelectedYear] = useState(ALL_DATES_VALUE);
   const [selectedDashboard, setSelectedDashboard] = useState(DEFAULT_DASHBOARD_GROUP);
```

Comment:
Perhaps this should start empty, now that the effect is taking care of setting the default if it exists

Reply:
Optional suggestion here @tcaiger

Reply:
I did do this initially and noticed a small visual delay before it sets the default. Since 9 times out of 10 the default is in the set, I though I might as well set it initially and have it slightly more seamless. I don't really mind though if you'd rather I unset it?

Reply:
No that makes sense! When we have other dashboards it might be worth revisiting, but for now that's perfectly reasonable :-)